<launch>
    <!-- GPS Reader Node -->
    <node name="gps_reader" pkg="data_bridge_ros1" type="gps_reader_node" output="screen">
        <param name="output_topic" value="/gps_imu_data" />
        <param name="shared_memory_key" value="G" />
        <param name="semaphore_key" value="g" />
        <param name="frame_id" value="gps_link" />
        <param name="publish_rate" value="10.0" />
    </node>
</launch>